docs/library/axi_adc_trigger/index.rst

```diff
@@ -0,0 +1,204 @@
+.. _axi_adc_trigger:
+
+AXI ADC Trigger
+================================================================================
+
+.. hdl-component-diagram::
+
+The :git-hdl:`AXI ADC Trigger <library/axi_adc_trigger>` IP core implements
+triggering for the ADC path and also controls two I/O triggering pins.
+
+More about the generic framework interfacing ADCs can be read here at :ref:`axi_adc`.
+
+Features
+--------------------------------------------------------------------------------
+
+-  AXI Lite control/status interface
+-  Analog triggers, two channels
+
+   -  Greater than a limit
+   -  Lower than a limit
+   -  Passing through the limit, high
+   -  Passing through the limit, low
+
+-  Digital triggers, two trigger pins
+
+   -  Low
+   -  High
+   -  Any edge
+   -  Rising edge
+   -  Falling edge
+
+-  Mixing analog and digital triggers
+-  Instrument trigger (from Logic Analyzer)
+-  Controls two IO trigger pins
+
+Files
+--------------------------------------------------------------------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Description
+   * - :git-hdl:`library/axi_adc_trigger/axi_adc_trigger.v`
+     - Verilog source for the peripheral.
+
+Block Diagram
+--------------------------------------------------------------------------------
+
+.. image:: block_diagram.svg
+   :alt: AXI ADC Trigger block diagram
+   :align: center
+
+Submodules
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  **Channel trigger**
+
+   -  Channel A
+
+      -  Amplitude limit - LIMIT_A (0x0014) - Defines the threshold level for
+         the ADC trigger
+      -  Function - TRIGGER_FUNCTION_A (0x0018) - Lower, higher than limit; pass
+         through limit
+      -  Hysteresis - TRIGGER_FUNCTION_A (0x001c) - "+-" value. Used for the
+         pass through functions
+
+   -  Channel B
+
+      -  Amplitude limit - LIMIT_A (0x0024) - Defines the threshold level for
+         the ADC trigger
+      -  Function - TRIGGER_FUNCTION_A (0x0028) - Lower, higher than limit; pass
+         through limit
+      -  Hysteresis - TRIGGER_FUNCTION_A (0x002c) - "+-" value. Used for the
+         pass through functions
+
+-  **External trigger**
+
+   -  Pin 0 (Ti)
+      Configurations - CONFIG_TRIGGER_I (0x004) allows for:
+
+      -  falling edge (bit 8)
+      -  rising edge (bit 6)
+      -  any edge (bit 4)
+      -  high level (bit 2)
+      -  low level (bit 0)
+
+   -  Pin 1 (To)
+      Configurations - CONFIG_TRIGGER_I (0x004) allows for:
+
+      -  falling edge (bit 9)
+      -  rising edge (bit 7)
+      -  any edge (bit 5)
+      -  high level (bit 3)
+      -  low level (bit 1)
+
+.. important::
+
+   In the case of M2k, the trigger pins (Ti and To) are chosen one
+   for input and one for output. This is for the ease of configuring a
+   daisy-chain of M2k's.
+   The hardware and software support both pins to be configured as input
+   or output in IO_SELECTION (0x000c).
```

Comment:
I don't think so, the verb is `support`.
Still will update to:
```
Both pins can be configured as input and output using the IO_SELECTION register (0x000c).
```
Since 'hardware and software' is pretty redundant.

Reply:
Ohhh yes, I just couldn't think of "support" as verb, even tho I read it maany times, thanks! :D